src/Hipace.cpp

```diff
@@ -59,59 +73,52 @@ Hipace::GetInstance ()
 }
 
 Hipace::Hipace () :
-    m_phys_const([this](){
-        m_instance = this;
-        amrex::ParmParse pph("hipace");
-        pph.query("normalized_units", this->m_normalized_units);
-        if (this->m_normalized_units){
-            return make_constants_normalized();
-        } else {
-            return make_constants_SI();
-        }
-    }()),
+    Hipace_early_init(this),
```

Comment:
It needed to be changed again because of necessity: `Parser::replaceAmrexParamsWithParser();` needs to be called before `amrex::AmrCore()` is initialized.